src/components/views/elements/TagTile.js

```diff
@@ -22,17 +22,36 @@ import sdk from '../../../index';
 import dis from '../../../dispatcher';
 import { isOnlyCtrlOrCmdKeyEvent } from '../../../Keyboard';
 
+import FlairStore from '../../../stores/FlairStore';
+
 export default React.createClass({
     displayName: 'TagTile',
 
     propTypes: {
-        groupProfile: PropTypes.object,
+        tag: PropTypes.string,
     },
 
     contextTypes: {
         matrixClient: React.PropTypes.instanceOf(MatrixClient).isRequired,
     },
 
+    componentWillMount() {
+        this.unmounted = false;
+        if (this.props.tag[0] === '+') {
+            FlairStore.getGroupProfileCached(
+                this.context.matrixClient,
+                this.props.tag,
+            ).then((profile) => {
+                if (this.unmounted) return;
+                this.setState({profile});
```

Comment:
`profile` needs adding to `getInitialState`, with a comment

Reply:
(this was done)